metrics2/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
This module should not exist.
Please move `metrics2` submodule to `$root/metrics`
The resulting structure should be:
metrics/metrics
metrics/metrics2
metrics/prometheus

The groupId is already following this, so the directory structure should as well.
Also Prometheus should not be touched by this change at all.